pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -4220,8 +4297,11 @@ def PumpCostRule(b, l1, l2):
         the cost of pumps.
         """
         # In the co_optimize method the objective should include the cost of pumps. So,
-        # delete the original objective to add a modified objective in this method.
-        del model.objective
+        # delete the original objectives to add a modified objective in this method.
+        del model.objective_Cost
```

Comment:
If you deactivate then why do we delete the objective from the model? I am not sure how exactly the objective function is working. But I believe if we deactivate things then we should not delete objectives from the model

Reply:
Good point - instead of deleting, I changed the code to deactivate the original objective functions.